app/assets/styles/bulletin.css

```diff
@@ -147,33 +147,43 @@ figure {
 }
 
 .featured-article__wrapper {
+	max-width: 78vw;
+	margin-left: auto;
+	margin-right: auto;
 	padding: 2em 3em 4em;
 }
 
 .featured-article__section-heading{
-	font-size: 2rem;
+  font-size: 2rem;
   font-weight: 300;
-  margin-bottom: 4rem;
+  margin-bottom: 3rem;
 }
 
 .featured-article__title {
-	font-size: 1.5rem;
-  line-height: 1.2;
+  font-size: 2rem;
+  line-height: 1.1;
   font-weight: 300;
-  margin-top: 0;
+  margin-top: .5rem;
 }
 
 @media(max-width: 680px) {
   .featured-article__title {
-    font-size: 1.75rem;
+    font-size: 1.6rem;
   }
 }
 
 .featured-article__abstract {
-	font-weight: 300;
-  line-height: 1.4;
+   font-weight: 300;
+   line-height: 1.4;
+   font-size: 1.2rem;
 }
 
+@media(max-width: 680px) {
+	.featured-article__abstract {
+	  font-size: 1rem;
+	}
+  }
```

Comment:
This bracket is not correctly in place. 

Did you install the `editorconfig` plugin?

Reply:
Resolved bracket issue.

No. Will do.